decidim-initiatives/app/views/decidim/initiatives/create_initiative/fill_data.html.erb

```diff
@@ -37,6 +48,10 @@
               <%= f.editor :description, lines: 8, toolbar: :full %>
             </div>
 
+            <div class="field">
+              <%= f.text_field :hashtag %>
+            </div>
+
             <% signature_type_options = signature_type_options(f.object) %>
```

Comment:
Just a last thing to move that I haven't seen before: 
I see that staring from this one, the next fields are: 
1. signature_type
2. scope
3. signature_end_date
4. area

Can we leave it as this 🙏🏽 ?
1. signature_type
2. signature_end_date
3. scope
4. area

It should be on both forms of course. The reasoning is that most of the times scopes and areas should be together.